Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/WindowsDictationInputDeviceManager.cs

```diff
@@ -59,11 +60,10 @@ public class WindowsDictationInputDeviceManager : BaseDeviceManager, IMixedReali
         /// </summary>
         private AudioClip dictationAudioClip;
 
-#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
-
         /// <inheritdoc />
         public override void Enable()
         {
+#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
```

Comment:
Let's revert all of this and just move the dictationRecognizer down under line 62

Reply:
Oh I see, that might not be possible bc the interface requires some of these methods.